### PR TITLE
MONA-92 setting up goals

### PR DESCRIPTION
**Pull request by mstone6769**

### CODE REVIEW

**Comment by mstone6769** [comment]
@acsearles this will get the progress bar on monarch so QA can test it

File: app/directives/goals/iui-activity-detail.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      <iui-stepper
+        ng-model="activity.progressMilestone"
+        min-value="0"
+        max-value="100"
+        increment-by="25"
+        stepper-id="stepper">
+      </iui-stepper>
+    </div>
+    <div class="col-sm-8">
+      <iui-progress-bar
```
Comment (acsearles) [inline app/directives/goals/iui-activity-detail.html]: Look there she is. She's so cute!

File: app/directives/goals/iui-activity-detail.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    function deleteActivity(activity) {
+      var index = $scope.goal.patientActivities.indexOf(activity);
+      if (index > -1) {
+        $scope.goal.patientActivities.splice(index, 1);
+        $location.path('/goals/detail/'+activity.goalId+'/');
+      }
+    }
+
+    function completeActivity(activity) {
```
Comment (acsearles) [inline app/directives/goals/iui-activity-detail.js]: Boy you really built this thing out. This is great.

File: app/services/goals-service.js [hunk 3]
```diff
@@ -1,7 +1,107 @@
 (function (app) {
   'use strict';
+
+  var progressBarOptions = {
```
Comment (acsearles) [inline app/services/goals-service.js]: Why are these not in a app.value()?

File: app/themes/default_sass/main.scss [hunk 4]
```diff
@@ -22,9 +22,20 @@
   text-transform: uppercase;
 }
 
+// Stylying of Progress bar - getting rid of Bootstrapfullness - if approved by the team - we can move this to the iui-general repo
```
Comment (acsearles) [inline app/themes/default_sass/main.scss]: I agree with these.

File: app/services/goals-service.js (hunk 3 above)
Comment (mstone6769) [inline app/services/goals-service.js]: they are repeated inside of the value, and I wanted to keep things DRY

**Pull request merged**